keras_vggface/constants.py

```diff
@@ -0,0 +1,14 @@
+V1_LABELS_PATH = 'https://github.com/rcmalli/keras-vggface/releases/download/v2.0/rcmalli_vggface_labels_v1.npy'
```

Comment:
What's the reason behind moving this from utils.py? 🔁

Reply:
Mainly because importing utils was clashing with actual tf.keras.utils. Instead of renaming utils to library_utils, it made more sense to just not import utils, and constants instead.

To avoid importing code that doesn't get used. The existing code uses "utils" as a place to dump everything. The utils functions are only used in examples, whereas the constant strings are used more often.